3rdParty/CMakeLists.txt

```diff
@@ -180,24 +185,6 @@ if (USE_IRESEARCH)
     "${IRESEARCH_ROOT}/cmake" # to find iResearch dependencies
   )
 
-  # cmake will seemingly randomly delete files from ${IRESEARCH_BUILD_DIR} during configuration
-  # so must copy required files only after the cmake configuration phase is complete, not via file(COPY ...)
-  # this command must _always_ be executed because the subsequent cmake configuration runs will once again delete files
```

Comment:
The cmake build process seems to sometimes remove files required for the IResearch 3rd-party library build process
therefore this workaround is in-place to ensure that the required files are always present when IResearch is built

Reply:
This patch will not copy files into the build directory at all. It simply will overrule their source file and the generation mechanism. 
Please provide a way to reproduce your claim that this doesn't work as intended.

Reply:
```
> Please provide a way to reproduce your claim that this doesn't work as intended.
remove the copying of files and try to rebuild, every so often, seemingly randomly, the build will fail due to these files missing from the directory
```
once the root cause of this issue is addressed then it is totally reasonable to remove file copying during the build stage, and only leave it for the configure stage